client/scripts/views/components/new_thread_form.ts

```diff
@@ -621,7 +621,9 @@ export const NewThreadForm: m.Component<{
                 defaultTopic: (vnode.state.activeTopic === false || vnode.state.activeTopic)
                   ? vnode.state.activeTopic
                   : localStorage.getItem(`${app.activeId()}-active-topic`),
-                topics: app.topics.getByCommunity(app.activeId()),
+                topics: app.topics.getByCommunity(app.activeId()).filter((t) => {
```

Comment:
I feel like a lot of these front-end checks for token thresholds should be scoped to only run iff we're in a token forum (which the app should know). We use components like `new_thread_form` in substrate communities as well as cosmos communities, so it probably doesn't make sense to run this kind of filter in the UI unless it's necessary.
Thoughts @jnaviask ?

Reply:
Also, when testing this component, when I didn't have any token in a wallet within the ALEX community (which didn't yet have a threshold set), it through the error `Unable to fetch user's token balance` from the threads controller.